sage-ios/SAGE/SAGE/ProfileCheckinTableViewCell.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  ProfileCheckinTableViewCell.swift
+//  SAGE
+//
+//  Created by Erica Yin on 12/4/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+
+import UIKit
+
+class ProfileCheckinTableViewCell: UITableViewCell {
```

Comment:
just make it CheckinTableViewCell, becuase we might wanna reuse this in other places. not just from the profile